spec/CloudCode.spec.js

```diff
@@ -19,11 +19,11 @@ const mockAdapter = {
 };
 
 describe('Cloud Code', () => {
-  it('can load absolute cloud code file', done => {
+  it('can load absolute cloud code file', (done) => {
```

Comment:
Do you think you can undo the lint changes? It would make it much faster to review the PR.

Reply:
done